resource.go

```diff
@@ -212,13 +235,35 @@ func (r *resource) Mode() os.FileMode {
 }
 
 func (r *resource) UID() string {
+	if r.user != "" {
+		u, err := user.Lookup(r.user)
```

Comment:
I think it's a good idea for Continuity to cache the results of these user and group lookups. The system library calls could be quite expensive when multipled by hundreds of thousands of files; I don't know.